build/configs/rtl8720e/rtl8720e_download.sh

```diff
@@ -50,7 +50,7 @@ function pre_download()
 		cp -p ${BIN_PATH}/${COMMON_BIN_NAME} ${IMG_TOOL_PATH}/${COMMON_BIN_NAME}
 	fi
 	if test -f "${SMARTFS_BIN_PATH}"; then
-		cp -p ${BIN_PATH}/rtl8720e_smartfs.bin ${IMG_TOOL_PATH}/rtl8721csm_smartfs.bin
+		cp -p ${BIN_PATH}/rtl8720e_smartfs.bin ${IMG_TOOL_PATH}/rtl8720e_smartfs.bin
 	fi
```

Comment:
You already check `${SMARTFS_BIN_PATH}` so that let's use it at cp command line as well.
```
cp -p ${SMARTFS_BIN_PATH} ${IMG_TOOL_PATH}/${CONFIG_ARCH_BOARD}_smartfs.bin
```